### DISCUSSION TITLE
A user has to log out from the WebUI first after added to a new team.

### DISCUSSION THREAD

**Discussion by evanchaoli** [concourse/concourse#5551] [General]
We have received plenty of queries that users complained that like:

```
$ fly --target runway-ci-stg login -c <url> -n <some-team>
logging in to team 'some-team'
navigate to the following URL in your browser:
  https://url/login?fly_port=65274
or enter token manually:
error: user [username] is not in team [some-team]
```

That because the user was in some other team and logged in to the WebUI already. As the WebUI stores authorization info in cookie, when user open fly promoted url in a web browser, it directly succeeds without typing in username/password, but the new team is not authorized.

We have to again and again tell new users to log out first, then re-login. Though we have documentation there, but it's really hard to expect new users to go through all documentations before they start to use Concourse.

So I think Concourse should enhance. For fly login URL, it should always ask for username/password and re-login. Or smartly determine the team has been stored in cookie.

**Comment by jamieklassen** [comment]
I think @pivotal-jwinters could comment, but I'm pretty sure this use case is precisely the big selling point of 6.1.0. It should be fixed by #4950, as discussed in the [release note](https://github.com/concourse/concourse/pull/4950/files?file-filters%5B%5D=.md#diff-d4becb815d0052a5c40f051b44a8e4f2R3-R8)!
